### ISSUE TITLE
Is this working as expected: # Example that uses the autograd for 2D fft: ??

### ISSUE THREAD

**Issue by Lokesh-Boominathan** [locuslab/pytorch_fft#10]
I tried running the example code for checking autograd. But print(fx.grad, fy.grad) gave me two matrices that had all 0s except for the first element which had a value of 100. 

Shouldn't the other values be non zero too? Or is what I got the expected output, if so how?

**Comment by riceric22** [comment]
My understanding is that the output happens because in the example, the backprop starts from a sum of the outputs of the FFT. The gradient thus passed to the backwards pass of the FFT function is a dense tensor of all 1s, which is one 'cycle' in the FFT space.

**Comment by Lokesh-Boominathan** [comment]
What did you mean by one cycle?

Isn't the sum affected by all values in the input, so shouldn't all the elements of input matrix have a gradient?
